DEVELOPING_OPAL/demos/src/main/scala/org/opalj/fpcf/analyses/FieldAccessInformationAnalysisDemo.scala

```diff
@@ -54,19 +60,25 @@ object FieldAccessInformationAnalysisDemo extends ProjectAnalysisApplication {
             case Some(requirements) => requirements + domain
         }
 
-        var propertyStore: PropertyStore = null
+        val propertyStore: PropertyStore = project.get(PropertyStoreKey)
         var analysisTime: Seconds = Seconds.None
         val analysesManager = project.get(FPCFAnalysesManagerKey)
+        val typeIterator = XTACallGraphKey.getTypeIterator(project)
+        project.updateProjectInformationKeyInitializationData(TypeIteratorKey) { _ => typeIterator }
+        project.updateProjectInformationKeyInitializationData(ContextProviderKey) { _ => typeIterator }
 
         time {
-            propertyStore = analysesManager
+            analysesManager
                 .runAll(
-                    EagerFieldAccessInformationAnalysis
+                    XTACallGraphKey.allCallGraphAnalyses(project)
+                        ++ Set(
+                            EagerFieldAccessInformationAnalysis,
+                            ReflectionRelatedFieldAccessesAnalysisScheduler
+                        )
                 )
-                ._1
             propertyStore.waitOnPhaseCompletion()
         } { t =>
-            analysisTime = t.toSeconds
+            analysisTime += t.toSeconds
```

Comment:
Why?

Reply:
Probably a residual from a time i ran two phases of computation